huaweicloud/data_source_huaweicloud_networking_port_v2.go

```diff
@@ -82,6 +82,22 @@ func DataSourceNetworkingPortV2() *schema.Resource {
 				Computed: true,
 				Elem:     &schema.Schema{Type: schema.TypeString},
 			},
+			"fixed_ips": {
```

Comment:
we get fetch the addresses from `all_fixed_ips`, so I think the attribute is redundant.